api/v1alpha1/constants.go

```diff
@@ -55,4 +55,5 @@ const (
 	ClusterControlPlaneAnnotations     = "tkg.tanzu.vmware.com/cluster-controlplane-endpoint"
 
 	AKODeploymentConfigControllerName = "akodeploymentconfig-controller"
+	ManagememtClusterRoleLabel = "cluster-role.tkg.tanzu.vmware.com/management"
```

Comment:
Could you run lint for this?

Reply:
who sets this label to cluster?

Reply:
I believe tkg cli will add this label to cluster

Reply:
Could you please run go-lint for this? Or for the whole project perhaps? Thanks

Reply:
Are you using goland or vs-code? If you are using goland, it is possible. Could you ensure it lint like following?:
```go
AKODeploymentConfigControllerName = "akodeploymentconfig-controller"
ManagememtClusterRoleLabel        = "cluster-role.tkg.tanzu.vmware.com/management"
```

nit: it is better to put it together with others here: https://github.com/vmware-samples/load-balancer-operator-for-kubernetes/blob/main/api/v1alpha1/constants.go#L13, since they are all TKG related constant vars

Reply:
I am using goland , I will make sure the format is correct